lms/djangoapps/instructor/tests/views/test_instructor_dashboard.py

```diff
@@ -345,6 +346,28 @@ def test_staff_can_see_writable_gradebook(self):
         'of enrolled learners.'
     )
 
+    @patch(
+        'lms.djangoapps.instructor.views.instructor_dashboard.settings.WRITABLE_GRADEBOOK_URL',
+        settings.LMS_ROOT_URL
+    )
+    def test_staff_can_see_writable_gradebook_as_subdirectory(self):
+        """
+        Test that, when the writable gradebook feature is enabled and
+        deployed in a subdirectory, a staff member can see it.
+        """
+        waffle_flag = waffle_flags()[WRITABLE_GRADEBOOK]
+        with override_waffle_flag(waffle_flag, active=True):
+            response = self.client.get(self.url)
+
+        expected_gradebook_url = '{}/{}'.format(settings.LMS_ROOT_URL, self.course.id)
```

Comment:
I think that in the tests we should use WRITABLE_GRADEBOOK_URL instead of LMS_ROOT_URL and if possible override the value of WRITABLE_GRADEBOOK_URL in order to include a subdirectory

```suggestion
        expected_gradebook_url = '{}/{}'.format(settings.WRITABLE_GRADEBOOK_URL, self.course.id)
```

Reply:
True! Applied this on the current commit.